packages/core/content-manager/shared/contracts/collection-types-v5.ts

```diff
@@ -0,0 +1,322 @@
+// TODO: Use this contract when we make the switch to the new V5 API
+import { errors } from '@strapi/utils';
+import { Schema, Common, Documents } from '@strapi/types';
```

Comment:
@jhoward1994 @joshuaellis I decided to create two new parallel -v5 file contracts for content manager. So when we decide to switch to the new api, we have this ready :) 

I left the original contracts as they are, as it is what we are actually returning at the moment.

Reply:
Good idea!!